cmd/commands.go

```diff
@@ -291,6 +291,17 @@ Downloads an updated Caddy binary with the same modules/plugins at the
 latest versions. EXPERIMENTAL: May be changed or removed.`,
 	})
 
+	RegisterCommand(Command{
+		Name:  "add-package",
+		Func:  cmdAddPackage,
+		Usage: "<packageName>",
+		Short: "Adds Caddy package (EXPERIMENTAL)",
+		Long: `
+Downloads an updated Caddy binary with new module/plugin.
+If plugin is already added - then return error.
+`,
```

Comment:
Maybe something like this:

```suggestion
		Short: "Add a package to Caddy (EXPERIMENTAL)",
		Long: `
Downloads an updated Caddy binary with the specified package (module/plugin)
added. Retains existing packages. Returns an error if the package is already
included. EXPERIMENTAL: May be changed or removed.
`,
```

Reply:
yeah, nice